docs/build-environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
   - conda-forge/label/jupyterlite_core_alpha
 
 dependencies:
-  - python=3.10
+  - micromamba
```

Comment:
Is this something we could / should document somewhere, for those experiencing the same issue?

Either in the docs, or a visible note in the README.